packages/payload/src/admin/components/elements/SortColumn/index.tsx

```diff
@@ -29,6 +29,8 @@ const SortColumn: React.FC<Props> = (props) => {
   const descClasses = [`${baseClass}__desc`]
   if (sort === desc) descClasses.push(`${baseClass}--active`)
 
+  const shouldRenderButtons = !disable && type !== 'array' && type !== 'group' && type !== 'blocks'
```

Comment:
In my opinion the SortColumn shouldn't have to know about the specifics between different types of fields. It already gets a `disabled` prop which should be enough to know whether to include the controls or not.

Reply:
@DanRibbens so `field.disableSort` is not a real prop, it really just needs to be removed - the condition would never be hit by anything.

However, I do agree that there should be just one source of truth - aka the `disable` prop

I've went ahead and updated the PR to essentially transferring the conditional to buildColumns and pass it into the disable prop there.

Let me know what you think.